src/iso19111/coordinateoperation.cpp

```diff
@@ -6183,6 +6183,38 @@ void Conversion::_exportToPROJString(
         formatter->addParam("o_lat_p", -southPoleLat);
         formatter->addParam("lon_0", southPoleLon);
         bConversionDone = true;
+    } else if (ci_equal(methodName, "Adams_Square_II")) {
+        // A bit hacky. Map directly from ESRI method and parameter names
+        // as we haven't defined a OGC-like mapping, and we have a fairly
+        // number of restrictions on the accepted values.
```

Comment:
Could you please rephrase this? It is unclear what you mean, I think there's either missing a word or a few too many

Reply:
hum, no, all words I intended are here :-) I'll try to make it clearer if I can...